sips/sip-017/sip-017-schnorr.md

```diff
@@ -0,0 +1,120 @@
+# SIP-017 — Schnorr signatures
+
+## Preamble
+
+**SIP:** 017 (awaiting acceptance)
+
+**Title:** Schnorr signatures
+
+**Author:** Asteria [asteria@syvita.org](mailto:asteria@syvita.org)
+
+**Consideration:** Technical
+
+**Type:** Consensus
+
+**Layer:** Consensus (soft fork)
+
+**Status:** Draft
+
+**Created:** 19 December 2021
+
+**License:** BSD 2-Clause
+
+**Sign-off:** —
+
+## Abstract
+
+This SIP describes the addition of Schnorr signature structure and validation to the account model for the Stacks blockchain.
+
+## Introduction
+
+The Stacks blockchain signing system was initially built on ECDSA, similar to how the Bitcoin blockchain was. However, it is widely acknowledged that Schnorr is the better signing algorithm over ECDSA. The primary reason it wasn’t used in the Bitcoin blockchain for many years is that Schnorr was patented until 2008, the same year Bitcoin was invented. At the time, Satoshi Nakomoto decided that Schnorr signatures lacked the popularity and testing required to secure a system as critical as Bitcoin.
+
+13 years later, Schnorr now has been tested and popularised through the Bitcoin Taproot upgrade in November 2021. Now that Schnorr has been implemented on Bitcoin, I believe we should adopt it for Stacks too.
```

Comment:
Phrasing: this is a technical document, so the first-person voice should not be used.

Also, I'd recommend dropping this paragraph.  If there are specific, principled reasons to add Schnorr signatures to Stacks' consensus rules, we should just use them to arguing for this change (like how you do below).